aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -158,6 +158,16 @@ public void run(String... args) {
             String requestedMimeType = requestHelper.getRequestedMimeType("result");
             String resultMime = requestedMimeType != null ? requestedMimeType : "application/x-netcdf";
 
+            HttpIndexReader indexReader = new HttpIndexReader(WpsConfig.getProperty(WpsConfig.GEOSERVER_CATALOGUE_ENDPOINT_URL_CONFIG_KEY));
+
+            //  If the transaction is a test transaction - we want to limit the temporal extent to a small (one time step) extent.
+            //  We'll just limit the test transaction to the latest timestep.
+            if(ExecuteRequestHelper.isTestTransaction(request)) {
+                String timestamp = indexReader.getLatestTimeStep(layer, "time");
+                logger.info("Last timestamp for layer [" + layer + "] = " + timestamp);
+                subset = overrideTestSubsetTimeRange(subset, timestamp);
```

Comment:
Would it be simpler to override after subsetting to avoid rewriting the passed parameters string?  e.g. down below after  

    CalendarDateRange subsetTimeRange = subsetParams.getTimeRange();

Reply:
I don't think that you can set properties on the SubsetParameters object after it is parsed.

Reply:
Ooops - got you now.  I'll look at overriding